src/modules/rppt_tensor_augmentations.cpp

```diff
@@ -467,84 +467,84 @@ rppt_blend_host(RppPtr_t srcPtr1,
     return RPP_SUCCESS;
 }
 
-/******************** color_jitter ********************/
+/******************** color_twist ********************/
+
+RppStatus
+rppt_color_twist_host(RppPtr_t srcPtr,
+                      RpptDescPtr srcDescPtr,
+                      RppPtr_t dstPtr,
+                      RpptDescPtr dstDescPtr,
+                      Rpp32f *brightnessTensor,
+                      Rpp32f *contrastTensor,
+                      Rpp32f *hueTensor,
+                      Rpp32f *saturationTensor,
+                      RpptROIPtr roiTensorPtrSrc,
+                      RpptRoiType roiType,
+                      rppHandle_t rppHandle)
+{
+    RppLayoutParams layoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
 
-// RppStatus
-// rppt_color_jitter_gpu(RppPtr_t srcPtr,
-//                       RpptDescPtr srcDescPtr,
-//                       RppPtr_t dstPtr,
-//                       RpptDescPtr dstDescPtr,
-//                       Rpp32f *alphaTensor,
-//                       Rpp32f *betaTensor,
-//                       RpptROIPtr roiTensorPtrSrc,
-//                       RpptRoiType roiType,
-//                       rppHandle_t rppHandle)
-// {
-// #ifdef OCL_COMPILE
```

Comment:
Why is this commented?

Reply:
I'll remove all the OCL_COMPILE for all the functions developed in a separate PR along with one other common API level change.